src/uct/api/uct.h

```diff
@@ -340,11 +340,29 @@ enum uct_md_mem_flags {
                                                 mapping may be deferred until
                                                 it is accessed by the CPU or a
                                                 transport. */
-    UCT_MD_MEM_FLAG_FIXED    = UCS_BIT(1)  /**< Place the mapping at exactly
+    UCT_MD_MEM_FLAG_FIXED    = UCS_BIT(1), /**< Place the mapping at exactly
                                                 defined address */
+    /* memory access flags */
+    UCT_MD_MEM_ACCESS_LOCAL         = UCS_BIT(2), /**< enable local read/write
```

Comment:
do we want to make it semetric ?  have local_read/write/atomic ?
As for now it looks odd.